tutorials/07-building-a-game-with-pygame.md

```diff
@@ -110,13 +112,20 @@ PyGame records all "events", including mouse clicks, and makes these available t
 
 If the user clicks on an empty space, that will still be recorded but we will simply ignore it. If the user clicks on a falling ball, we want it to change direction.
 
-Right after the `while True` loop, add the following lines of code.
+To achieve this, add a `for` loop inside the existing `while` loop. The entire game loop should look as follows:
 
 ```python
+    while True:
         for event in pygame.event.get():
             if event.type == pygame.MOUSEBUTTONDOWN:
                 if ball.rect.collidepoint(pygame.mouse.get_pos()):
-                    ball.speed = [0,-1]
```

Comment:
Double-check the code here. Check if in sequence the ` ball.speed = [0, -1]` should be there and the below two lines removed rather?

Reply:
Fixing this error by replacing the two ball.speed lines with ball.speed[0,-1]